controllers/fenceagentsremediation_controller.go

```diff
@@ -48,6 +48,11 @@ func (r *FenceAgentsRemediationReconciler) SetupWithManager(mgr ctrl.Manager) er
 		Complete(r)
 }
 
+const (
+	methodNameFA      = "--action"
```

Comment:
can we have more descriptive names please? e.g. `parameterNameAction` and `parameterValueActionReboot`?